### ISSUE TITLE
[🐛 Bug]: Sorting and searching Sessions in the Selenium Grid's web interface only only applies to the displayed page

### ISSUE THREAD

**Issue by arnonax-tr** [SeleniumHQ/selenium#11607]
### What happened?

The Sessions tab in the Grid's web interface allows to sort by any of the columns, and also to search for a session using a textbox. However, if there are more sessions than what's displayed in one page, then these operations only operate on the content of the current page.

### How can we reproduce the issue?

```shell
* Open the web interface of Selenium Grid with many active sessions (more than one page)
* Click on the Sessions menu item (or go directly to `/ui#/sessions`)
* Sort by one of the columns, or use the Search box to search for a session ID that exists in **another page**
**Actual result:**
* Only the displayed sessions are sorted, and searching for a session that exists in another page will result in an empty search results.
**Expected result:**
The sorting and searching will apply to all active sessions (i.e. the search and sort should be done on the server side and not on the client side)
```

### Relevant log output

```shell
N/A
```

### Operating System

Linux, docker

### Selenium version

Irrelevant

### What are the browser(s) and version(s) where you see this issue?

Irrelevant

### What are the browser driver(s) and version(s) where you see this issue?

Irrelevant

### Are you using Selenium Grid?

4.8.0 (revision 267030adea)

**Comment by arnonax-tr** [comment]
AFAICT this issue is still relevant. Why didn't anyone looked at it yet? If you need more information please let me know.

**Comment by arnonax-tr** [comment]
This is not a very critical bug, but it's still relevant.

**Comment by VietND96** [comment]
@arnonax-tr, Can you give few screenshots with your reproduce steps? Since I could not reproduce the issue, for example below screenshots, sessions table with pagination, a session not present in first page, it could be seen after searching the ID.

![image](https://github.com/user-attachments/assets/b6e3455a-9230-4246-b455-af13db79eed8)
![image](https://github.com/user-attachments/assets/7efffbfb-fa61-43a9-b3ea-bdeea64f0c25)

**Comment by arnonax-tr** [comment]
Sorry, I also tried to reproduce it now and it seems to work fine. I don't remember why or where I saw it. The only thing I see now is that sorting by duration doesn't work well, but it also doesn't seem to be related to the paging, but rather to the fact that it sorts it as a string and not as a number:
![image](https://github.com/user-attachments/assets/073d1421-cf02-4166-b5ea-87d51061ca72)
Maybe that's what I saw in the first place and I jumped into wrong conclusions about the cause.
